### PR TITLE
OSDOCS-9333:updates config for RHOSP

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9333

Link to docs preview:
https://71647--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/move-etcd-different-disk.adoc [hunk 1]
```diff
@@ -64,7 +64,7 @@ spec:
           [Service]
           Type=oneshot
           RemainAfterExit=yes
-          ExecStart=/usr/lib/systemd/systemd-makefs xfs /dev/<new_disk_name>
+          ExecStart=/usr/lib/systemd/systemd-makfs.xfs -f /dev/vdb<new_disk_name>
```
Comment (EmilienM) [inline modules/move-etcd-different-disk.adoc:67]:
```suggestion
          ExecStart=/usr/lib/systemd/systemd-makfs.xfs -f /dev/<new_disk_name>
```

Why adding vdb?

**Comment by JoeAldinger** [comment]
PTAL @geliu2016

**Review by skopacz1 — APPROVED** [review]
LGTM

**Pull request merged**